pom.xml

```diff
@@ -375,6 +369,12 @@ under the License.
       <version>${mavenResolverVersion}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
+      <artifactId>wiremock-standalone</artifactId>
```

Comment:
I'm hesitant to introduce yet another mocking framework to the codebase for complexity reasons. Any chance we could simply use Mockito?

Reply:
drop from test dependencies ... used only in IT